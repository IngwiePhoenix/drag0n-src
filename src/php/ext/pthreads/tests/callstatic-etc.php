<?php
class Test {
    public static function __callStatic ($name, $args) {
        var_dump($name, $args);
    }
}

class UserThread extends Thread {
    public function run () {
        Test::called_func("argument");
    }
}

$thread = new UserThread;
$thread->start();
?>
