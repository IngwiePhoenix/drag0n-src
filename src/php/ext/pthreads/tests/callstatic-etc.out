Warning: Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) in Unknown on line 0
string(11) "called_func"
array(1) {
  [0]=>
  string(8) "argument"
}